File: sdkwork-ai-sdk-csharp/Models/RunStepListResponse.cs
using System;
using System.Collections.Generic;
using System.Text.Json.Serialization;

namespace Ai.Models
{
    public class RunStepListResponse
    {
        public string? Object { get; set; }
        public List<RunStepResponse>? Data { get; set; }
        public bool? HasMore { get; set; }
        public string? FirstId { get; set; }
        public string? LastId { get; set; }
    }
}

File: sdkwork-ai-sdk-python/sdkwork_ai_sdk/models/run_step_list_response.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class RunStepListResponse:
    """OpenAI run step list response"""
    object: str = None
    data: List[RunStepResponse] = None
    has_more: bool = None
    first_id: str = None
    last_id: str = None
